tox.ini

```diff
@@ -1,5 +1,7 @@
 [tox]
 envlist = py37, py38, py39, lint
+skip_missing_interpreters = True
```

Comment:
lowercase t ?

Reply:
@tlambert03 bump, does this need to be lowercased? If not then all good and will merge now

Reply:
thanks!